parm/stage/stage.yaml.j2

```diff
@@ -0,0 +1,99 @@
+{% set cycle_HH = current_cycle | strftime("%H") %}
```

Comment:
Some documentation up-top will be nice.
Something like:
```suggestion
# This yaml is intended to be of the following structure:
key:
    mkdir:
        - "COM directory to create"
    copy:
        - ["source_file", "destination_file"]
# Jinja is permitted in this yaml, as long as the keys are:
# - COM_
# - DO_ATM, DO_OCN, DO_ICE
# For a full list see exglobal_stage_ic.py        

{% set cycle_HH = current_cycle | strftime("%H") %}
```

Reply:
Great suggestion, thanks a bunch!